### PR TITLE
added more info section, which now contains browser compatibility and…

### PR DESCRIPTION
**Pull request by seewah**
# Problem

Official policy not clear

# Solution

Clear statement in readme

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [X] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [n/a] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-325-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-325-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://325-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-325-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://325-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md [hunk 1]
```diff
@@ -359,12 +359,21 @@ A few things to check before you go live:
...
+
+### Browser compatibility
 
 ![Chrome](https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png) | ![Firefox](https://raw.githubusercontent.com/alrra/browser-logos/master/src/firefox/firefox_48x48.png) | ![IE](https://raw.githubusercontent.com/alrra/browser-logos/master/src/edge/edge_48x48.png) | ![Safari](https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari/safari_48x48.png)
 --- | --- | --- | --- |
 Latest ✔ | Latest ✔ | 11+ ✔ | Latest ✔ |
 
+### Support
+
+Please open an issue through [GitHub](https://github.com/onfido/onfido-sdk-ui/issues). Please be as detailed as you can. Remember **not** to submit your token in the issue. Also check the closed issues to check whether it has been previously raised and answered.
```
Comment (stefaniacardenas) [inline README.md]: ` Also check the closed issues to check` sounds a bit redundant. Would one of these two options read better? `Also check the closed issues to find out [...]` or `Also check the closed issues to make sure that the issue hasn't been previously raised and answered`

Comment (seewah) [reply README.md]: thanks. fair comment. I copied this from iOS. However, to be honest, at this stage, i think I will keep it as it is, as the other two PRs have been approved.

**Pull request merged**